### ISSUE TITLE
Incompatibility with Wordpress 5.3 and 5.4 (not present in 5.5+)

### ISSUE THREAD

**Issue by samueljseay** [woocommerce/woocommerce-admin#6194]
While building the new unit test matrix in #6164 I noticed a consistent failure in matrix runs of Wordpress 5.3 and 5.4.

The stack trace for the test failure:

```
3) WC_Tests_API_Reports_Orders::test_custom_product_attributes_filter
Undefined index: items
/tmp/wordpress/wp-includes/rest-api.php:1237
/tmp/wordpress/wp-includes/rest-api.php:1237
/tmp/wordpress/wp-includes/rest-api.php:1037
/tmp/wordpress/wp-includes/rest-api/class-wp-rest-request.php:885
/tmp/wordpress/wp-includes/rest-api/class-wp-rest-server.php:884
/tmp/wordpress/wp-content/plugins/woocommerce/tests/legacy/framework/vendor/class-wp-test-spy-rest-server.php:35
/tmp/wordpress/wp-content/plugins/woocommerce-admin/tests/api/reports-orders.php:303
```

It seems this is a real incompatibility that needs investigating and fixing. But first we need to reproduce it in a local dev site. To move forward the unit test migration we've opted to leave 5.3 and 5.4 out of our test runs until we can fix this.

**Comment by moon0326** [comment]
This is going to be very tricky to fix.

WP 5.3 [rest-api.php](https://github.com/WordPress/WordPress/blob/2cfd371ed4d41a47e2211fce792291c42fd7d02e/wp-includes/rest-api.php#L1236) requires `items` to be defined when the `type` is array, whereas [rest-api.php ](https://github.com/WordPress/WordPress/blob/537fd931bc02e6e934a2d774422b897871aa87ad/wp-includes/rest-api.php#L1595)in 5.5+ does not by checking if `$args['items']` is set.

Since the test checks both [string and array](https://github.com/woocommerce/woocommerce-admin/blob/a39fc87ffeb1ec78c1766a625ac0a09da7aa7086/tests/api/reports-orders.php#L178), it's hard to define a correct schema. 

We can spend some time updating the test, but I think we can also skip the test for 5.3 and 5.3. @samueljseay  What do you think?

**Comment by adrianduffell** [comment]
@moon0326 Thanks for grabbing this!

It sounds like this is a validation bug in WP 5.3 that was fixed in 5.5? If so it seems fair to skip the failing tests in older versions of WP.

**Comment by moon0326** [comment]
Thank you @adrianduffell. I will make a PR to skip those two failing tests.
